modules/ocf_stats/manifests/prometheus.pp

```diff
@@ -1,8 +1,21 @@
 # prometheus daemon config
 class ocf_stats::prometheus {
+  file {
+    '/usr/local/bin/gen-prometheus-nodes':
+      source => 'puppet:///modules/ocf_stats/prometheus/gen-prometheus-nodes',
+      mode   => '0755';
+  }
+
+  cron { 'gen-prometheus-nodes':
+    command => '/usr/local/bin/gen-prometheus-nodes /var/local/prometheus-nodes.json',
+    user    => 'root',
```

Comment:
I think the default value is root, so the line isn't strictly necessary.

Reply:
Again this was taken from the munin configuration, but I'll definitely change this one because it seems silly.